recipe/meta.yaml

```diff
@@ -48,16 +48,22 @@ requirements:
 {% set tests_to_skip = tests_to_skip + " or test_loadings_converges" %}
 # Numerically unstable test numerical difference in test
 {% set tests_to_skip = tests_to_skip + " or test_mlp_regressor_dtypes_casting" %}         # [ppc64le]
+# On win32 there are failed tests:
+#{% set tests_to_skip = tests_to_skip + " or test_precomputed_nearest_neighbors_filtering or test_precomputed_nearest_neighbors_filtering" %}         # [win32]
+#{% set tests_to_skip = tests_to_skip + " or test_estimators[GaussianProcessRegressor()-check_supervised_y_2d] or test_estimators[GaussianProcessRegressor()-check_fit_idempotent]" %}         # [win32]
+#{% set tests_to_skip = tests_to_skip + " or test_estimators[Nystroem()-check_fit_idempotent] or test_family_bounds[family0-expected0]" %}         # [win32]
```

Comment:
If these are not being skipped as expected, we could try [this approach](https://github.com/AnacondaRecipes/scikit-learn-feedstock/commit/ab8a2daad214491c6e39888cb3c84bc31f62c28c#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a) (from v0.24.2).

Reply:
Thank you, Anthony. After skipping failed tests it succeded on win32